web_select_all_companies/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 {
     "name": "Web Select All Companies",
     "summary": "Allows you to select all companies in one click.",
-    "version": "16.0.1.0.1",
+    "version": "15.0.1.0.1",
```

Comment:
This should be part of the migration commit

Reply:
Fixed @BT-anieto